apps/src/templates/BaseDialog.jsx

```diff
@@ -156,6 +150,16 @@ export default class BaseDialog extends React.Component {
       }),
       ...this.props.style,
     };
+    xCloseStyle = {
```

Comment:
I'm reading this correctly that it's all 'new' because it lost an indent? I know some of the styling is actually new too:)

Reply:
Yes! I updated the padding and border in an attempt to keep the dialog overall looking the same but am happy to include those back in!

Reply:
No no the styling looks good! Just was confirming why all of it is highlighted green/new.